src/services/tourettes.js

```diff
@@ -62,12 +62,24 @@ function getProfanities(amount) {
   return profanityReply.join(' ');
 }
 
+function isInt(data) {
+  if (typeof data === 'number' && (data % 1) === 0) {
```

Comment:
Aah, that's the syntax I was looking for, I tried return if(...); first and it failed and I sort of gave up.